logic.js

```diff
@@ -0,0 +1,221 @@
+MAINAPP =(function(){
+
+    const url ="https://raw.githubusercontent.com/alexsimkovich/patronage/main/api/data.json"
+   
+    const pizzasSpace = document.querySelector("#pizzaSpace");
+    const chartSpace = document.querySelector("#chartSpace");
+    const price_sum = document.querySelector("#price_sum");
+    const empty_chart_text = document.querySelector("#empty_chart_text");
+    
+    price_sum.style.display="none";
+
+    const chart =[];
+
+    const data = JSON.parse(GetData(url));
```

Comment:
nazwij to inaczej, `data` jest za ogolnie

Reply:
nazwa zmieniona na pizzaData